package-lock.json

```diff
@@ -40,7 +39,7 @@
         "@types/node": "^16.18.25",
         "@types/react": "^18.2.0",
         "@types/react-dom": "^18.2.1",
-        "@wormhole-foundation/wormhole-connect": "^0.1.4",
+        "@wormhole-foundation/wormhole-connect": "file:../wormhole-connect/wormhole-foundation-wormhole-connect-0.0.1-beta.0.tgz",
```

Comment:
Should we change this before merging or is it needed until a new version of wormhole-connect with this PR's changes is published?

Reply:
didnt mean to include this